app/helpers/admin/volunteers_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+  module VolunteersHelper
+    def can_manage_volunteers?(conference)
+      if (current_user.has_role? :organizer, conference) || (current_user.has_role? :volunteers_coordinator, conference)
+        true
+      else
+        false
+      end
```

Comment:
It seems I wrote a comment a few days, but never actually sent it.
We can rewrite this as:
```ruby
def can_manage_volunteers?(conference)
  current_user.has_role?(:organizer, conference) || current_user.has_role?(:volunteers_coordinator, conference)
end
```